modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/user/selfRegister/self-register-request-body.json

```diff
@@ -25,7 +25,7 @@
   "properties": [
     {
       "key": "callback",
-      "value": "https://localhost:9443/myaccount/login"
+      "value": "https://localhost:9853/myaccount/login"
```

Comment:
Why do we need to change this

Reply:
Please refer the 2nd point of the description.